src/NuGet.Core/NuGet.Packaging/Signing/Utility/X509ChainHolder.cs

```diff
@@ -41,4 +41,4 @@ public void Dispose()
             }
```

Comment:
Undo changes to this file.

Reply:
Since insert_final_newline = true in NuGet.Client/.editorconfig. It adds a new line by default when I save it.

Reply:
Again, don't save.  Exclude the file from your PR.